offchain/dusd-ui/assets/styles/light.scss

```diff
@@ -0,0 +1,120 @@
+@charset "utf-8";
+
+@import "../../node_modules/bulma/sass/utilities/initial-variables";
+
+@font-face {
+  font-family: "Brandon Grotesque";
+  src: local("Brandon Grotesque Light"), local("Brandon-Grotesque-Light"),
+    url(../fonts/Brandon-Grotesque/brandon-grotesque-light.woff2)
+      format("woff2");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Brandon Grotesque";
+  src: local("Brandon Grotesque Bold"), local("Brandon-Grotesque-Bold"),
+    url(../fonts/Brandon-Grotesque/brandon-grotesque-bold-regular.woff2)
+      format("woff2");
+  font-weight: 700;
+}
+
+@font-face {
+  font-family: "Brandon Grotesque";
+  src: local("Brandon Grotesque Black"), local("Brandon-Grotesque-Black"),
+    url(../fonts/Brandon-Grotesque/brandon-grotesque-black.woff2)
+      format("woff2");
+  font-weight: 900;
+}
+
+$scheme-main: #f5fcfe;
+$scheme-invert: #080e42;
+
+$family-primary: "Brandon Grotesque Bold", serif;
+
+$link: $scheme-invert;
+$text: $scheme-invert;
+$link-focus: $scheme-invert;
+$link-hover: $scheme-invert;
+$input-color: $scheme-invert;
+$text-strong: $scheme-invert;
+$title-color: $scheme-invert;
+$subtitle-color: $grey-light;
+$button-border-width: 0;
+
+@import "../../node_modules/bulma/bulma";
+
+.navbar-item.is-active {
+  border-bottom: 3px solid #2e49c5;
+}
+
+.dusd-btn-has-background {
+  background: linear-gradient(89.62deg, #030d53 0.3%, #3b4bc2 99.64%);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 115%;
+  text-align: center;
+  color: #ffffff;
+}
+
+.dusd-btn-has-border {
+  border: 3px solid rgba(30, 39, 105, 1);
+  box-shadow: 0px 0px 20px 5px rgba(45, 59, 160, 0.4);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 100%;
+  text-align: center;
+  color: #1e2769;
+}
+
+.navbar-brand a.navbar-item {
+  background-image: url(../images/logo_header_light.png);
+  background-size: cover;
+  width: 65.92px;
+  height: 60px;
+}
+
+#lightmode-toggle-btn {
+  background-image: url(../images/lightmode_toggle_btn_light.png);
+  background-size: cover;
+  width: 96px;
+  height: 50px;
+}
+
+#home-static {
+  background-image: url(../images/home_static_light.png);
+  background-size: cover;
+}
+
+.vaults-table-row {
+  border-bottom: 1px solid #3651cd;
+}
+
+.link-a-coin {
+  background: linear-gradient(179.9deg, #0d1e7e 0.1%, #4d5ec1 99.98%);
```

Comment:
though I don't think the users of the website will use outdated browsers or text only browser like lynx since we need to the users to connect to their wallets, but I do agree this is a good practice to follow

Reply:
You'd be surprised at what situations you could run into--like some bug in the browser or GPU that can't do gradients. Because it doesn't lose us much, it can also helps encourage compatibility for new and obscure browsers.... not just TUIs! Netsurf, Dillo can render the background color, and Ladybird will forever be on my watch now.

Reply:
More this is just a good habit to consider.